File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cs
public class Solution {
    public int FindMaxConsecutiveOnes(int[] nums) {
        int max=0;
        int count=0;
        foreach(int val in nums){
            if(val==1){
                count++;
                max=Math.Max(max,count);
            }else{
                count=0;
            }
        }return max;
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int count=0;
        int max=0;
        for(int j=0;j<nums.length;j++){
            if(nums[j]==1){
                count++;
            }
            max=Math.max(max,count);
            if(nums[j]==0)count=0;
        }
        return max;
    }
}